settings/LoanPolicy/components/EditSections/RequestManagementSection.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import {
+  Checkbox,
+  Accordion,
+} from '@folio/stripes/components';
+
+import { intervalPeriods } from '../../../../constants';
+import PolicyPropertySetter from '../PolicyPropertySetter/PolicyPropertySetter';
+
+
+class RequestManagementSection extends React.Component {
+  static propTypes = {
+    policy: PropTypes.object.isRequired,
+  };
+
+  render() {
+    const { policy } = this.props;
+
+    if (!policy.loanable) {
+      return null;
+    }
+
+    return (
+      <React.Fragment>
+        <h2>
+          <FormattedMessage id="ui-circulation.settings.requestManagement.requestManagement" />
+        </h2>
+        <Accordion
+          label={<FormattedMessage id="ui-circulation.settings.requestManagement.recalls" />}
+        >
+          <PolicyPropertySetter
+            fieldLabel="ui-circulation.settings.requestManagement.recallReturnInterval"
+            selectPlaceholder="ui-circulation.settings.loanPolicy.selectInterval"
+            inputValuePath="requestManagement.recalls.recallReturnInterval.duration"
```

Comment:
Is this structure agreed with BE? `requestManagement.recalls.recallReturnInterval.duration`